curvefs/src/client/fuse_client.h

```diff
@@ -58,6 +58,8 @@
 #include "src/common/throttle.h"
 #include "curvefs/src/client/filesystem/meta.h"
 #include "curvefs/src/client/filesystem/filesystem.h"
+#include "curvefs/src/client/fsused_updater.h"
+#include "curvefs/src/client/fsquota_checker.h"
```

Comment:
ditto, put includes where you directly need them

Reply:
fix